src/org/opendatakit/briefcase/ui/PushTransferPanel.java

```diff
@@ -169,7 +172,7 @@ public void actionPerformed(ActionEvent e) {
   /**
    * Create the transfer-from-to panel.
    *
-   * @param txtBriefcaseDir
+   * @param terminationFuture
```

Comment:
I know you didn’t create it, but these @params with no explanations have no value and should be deleted.

Reply:
Parameter name was wrong so I just corrected it. Will remove it.

Reply:
Thanks. What you first did was reasonable.